pages/electronics-library.tsx

```diff
@@ -296,24 +320,25 @@ const Home = ({
 
       <div className={styles.electronicsContainer}>
         {filteredComponents.map((item: any) => (
-          <div
-            key={item.name + item.shortDescription.substring(0, 30)}
-            className={styles.electronicsItem}
-            onClick={() => {
-              setShowOverlay(true);
-              setActiveProduct(item);
-            }}
-          >
-            <div className={styles.productName}>{item.name}</div>
-            <img src={item.productPhoto[0]} alt={item.name} />
+          <Link href={`/electronics-library#${item.id}`} key={item.id + item.shortDescription.substring(0, 30)}>
```

Comment:
We will run into the issue of two items having the same id again.

Reply:
In the spreadsheet someone added a rule so duplicates are being highlighted, so there is some effort to prevent them. We might be okay here.
![image](https://user-images.githubusercontent.com/129784422/234702094-bc732933-b418-4d7c-87c8-a6336d632512.png)

Reply:
Yeah, I added that rule. @poofytoo you'll have to make sure there are no duplicates on that sheet.